packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -1,14 +1,108 @@
+
+---
+
+Apex Replay Debugger helps you debug issues in an after-the-fact manner. Your debugging process is a repetition of editing your Apex code, pushing or deploying the code to your org, reproducing the buggy scenario, downloading the resulting debug log, and launching Apex Replay Debugger with that debug log.
 
 ## Prerequisites
+Before you set up Apex Replay Debugger, make sure that you have these essentials.
+
+* **Salesforce CLI**  
+  Before you use Salesforce Extensions for VS Code, [set up Salesforce CLI](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup). 
+* **A Salesforce DX project**  
+  See [Project Setup](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_workspace_setup.htm) in the _Salesforce DX Developer Guide_ for details.
+* **An active default scratch org _OR_ a local copy of a debug log from the org whose up-to-date source is in your Salesforce DX project**
+  1. To create a scratch org, you need a Dev Hub. For information on setting up your production org as a Dev Hub, see [Enable Dev Hub in Your Org](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_enable_devhub.htm) in the _Salesforce DX Setup Guide_.  
+  1. To authorize your Dev Hub, open VS Code’s command palette (Cmd+Shift+P on macOS, or Ctrl+Shift+P on Windows or Linux) and run **SFDX: Authorize a Dev Hub**.  
+  1. To create a default scratch org, run **SFDX: Create a Default Scratch Org**. Then, run **SFDX: Push Source to Default Scratch Org**. For information about scratch orgs, see [Scratch Orgs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs.htm) in the _Salesforce DX Developer Guide_.
+* **[Visual Studio Code](https://code.visualstudio.com/download) v1.17 or later** 
+* **The latest versions of the [salesforcedx-vscode-core](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-core) and [salesforcedx-vscode-apex](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex) extensions**  
+We suggest that you install all extensions in the [salesforcedx-vscode](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) extension pack.
+
+## Set Up Apex Replay Debugger
+The first time that you use Apex Replay Debugger, create a launch configuration. Each time that you debug an issue, set up an Apex Replay Debugger session.
```

Comment:
Ruth, these two sentences read a bit awkwardly, to me. The first sentence should probably be something like: "The first time that you use Apex Replay Debugger you will need to create a launch configuration." The second sentence "Each time that you debug an issue, set up an Apex Replay Debug session." I don't exactly understand what we're trying to convey here and it feels like something small is missing.

Reply:
Thanks for the feedback, @JimSuplizio . A couple of things:
* Each section of our docs is supposed to start with a summary of the key points in that section, so users know what to expect. This is an intro for the three `###`-level sections below. (I just noticed that there's no intro for the section on launch configurations, so I'll add one.)
* Our style guidelines say to avoid "need to", using imperative verbs instead. Our style guidelines also say to avoid future-tense verbs.

Given those things, do you still think changes are needed to these two sentences?

Reply:
They still read awkwardly but I'll get over it. The second sentence still feels like some fact or other instruction is missing, or even that it's unnecessary. "Each time that you debug an issue, setup up an Apex Replay Debugger session" as opposed to doing what else?

Reply:
I added a "Then, " at the beginning of the second sentence in the commit that I just pushed. I want to explain that the "Create a Launch Configuration" steps need to be done only once, but that the "Set Up an Apex Replay Debugger Session for a Scratch Org" or "Set Up an Apex Replay Debugger Session for a Sandbox or Production Org" steps will be repeated. Hopefully the "Then, " will help. If it doesn't, hopefully customers will submit feedback.

Reply:
That's fine. Approving now.